apps/docs/pages/guide/create-commandkit.mdx

```diff
@@ -12,14 +14,28 @@ npm create commandkit@latest
 
 This will start the CLI in the current directory.
 
+### Development version
+
+Similar to the main package, you could try the `dev` or `next` versions of create-commandkit:
+
+<Callout type="warning">
+  This version is likely to have issues. Be careful!
+</Callout>
+
+```sh copy
```

Comment:
Would `npm2yarn` work here? Perhaps it could make it easier for other package managers.
Update: Nevermind. Nextra currently doesn't support this for `create`